schemas/computation.schema.tpl.json

```diff
@@ -0,0 +1,84 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/Computation",
+    "_extends": "../../core/research/protocolExecution.schema.tpl.json",
```

Comment:
@lzehl, any idea how to reference this? We never actually extended anything across repos before...

Reply:
We do not support this momentarily, but there are good reasons to support this and it should not be too tricky to adapt the openMINDS generator accordingly. Technical details need to be discussed with Oli.